pkg/autoscaler/autoscaler.go

```diff
@@ -344,6 +344,17 @@ var _ = Describe("Autoscaler should", framework.LabelAutoscaler, Serial, func()
 					workloadArch = arch
 					expectedScaledMachineSet = machineSet
 				}
+
+				It("should have expected upstream annotations", func() {
```

Comment:
we talked about this at standup and think the `It` might be causing issues. try changing it to 

```suggestion
				By("should have expected upstream annotations", func() {
```